clang/test/SemaSYCL/unique_stable_name.cpp

```diff
@@ -0,0 +1,119 @@
+// RUN: %clang_cc1 %s -fsyntax-only -ast-dump -fsycl-is-device -triple spir64 -verify
```

Comment:
Why the `-ast-dump`? Also, is there a need for the `-triple`?

Reply:
No need for either. Will remove.